client/src/store/modules/projects/getters.ts

```diff
@@ -50,7 +50,7 @@ export const getters: GetterTree<IProjectState, {}> = {
   [PROJECT_FILTER](state, projectGetters) {
     const addons: { [key: string]: any } = projectGetters[ADDONS];
 
-    return Object.keys(addons).map(key => {
+    let models = Object.keys(addons).map(key => {
```

Comment:
Why variable?

Reply:
was required for first approach, forgot to remove - will fix